azurerm/resource_arm_loadbalancer.go

```diff
@@ -81,6 +81,13 @@ func resourceArmLoadBalancer() *schema.Resource {
 							ValidateFunc: azure.ValidateResourceIDOrEmpty,
 						},
 
+						"public_ip_prefix_id": {
```

Comment:
could we add this new field [to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/loadbalancer.html.markdown#L57)?